src/main/frontend/handler/editor.cljs

```diff
@@ -2696,7 +2696,8 @@
             (delete-concat current-block)))
 
         :else
-        (delete-and-update input current-pos (inc current-pos))))))
+        (delete-and-update 
+          input current-pos (util/safe-inc-current-pos-from-end (.-value input) current-pos))))))
```

Comment:
safe-inc-current-pos-from-end is not defined.

Reply:
resolved by fix typo.
The define of function can be found in
https://github.com/logseq/logseq/blob/e8bf41959663c7a52adf959f3d4c565e64294dec/src/main/frontend/util.cljc#L675-L687
and the test about this function can  be found in
https://github.com/logseq/logseq/blob/e8bf41959663c7a52adf959f3d4c565e64294dec/src/test/frontend/util_test.cljs#L11-L23